Libs/MRML/Core/vtkMRMLSegmentationNode.cxx

```diff
@@ -460,8 +460,16 @@ void vtkMRMLSegmentationNode::GetRASBounds(double bounds[6])
     {
     this->Segmentation->GetBounds(bounds);
     }
+
 }
 
+//---------------------------------------------------------------------------
+void vtkMRMLSegmentationNode::GetNodeBounds(double bounds[6])
+{
+  Superclass::GetNodeBounds(bounds);
```

Comment:
Could you rename this method to GetBounds()?
node->GetBounds() is more compact and readable than node->GetNodeBounds() and it's also consistent with the corresponding VTK method name.